Resources/Views/base.leaf

```diff
@@ -16,7 +16,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/img/favicon-16x16.png">
 
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
-    <link rel="stylesheet" href="css/style.css"/>
+    <link rel="stylesheet" href="/css/style.css"/>
```

Comment:
Interesting that this had to change - does it throw a 404 on the live site?

Reply:
This actually makes total sense, these are in /something rather than root so they'd look for styles in their own root and fail without this change. Nice spot!

Reply:
This was causing a problem for me once I created edit form, hopefully I haven't broke anything. 😬 

Though I think it shouldn't change anything, and it was happening cause the resource lives at the root of the web site, whereas without this going to the presentations page would try and look for it at `swiftleeds.co.uk/admin/presentations/styles.css` which doesn't exist.